src/components/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -327,7 +335,15 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
 
 .v-list-item {
   min-height: 0;
-  padding: 0 1rem 0 0.5rem;
+  padding: 0.5rem 1rem;
+}
+
+.v-list-item__subtitle {
+  color: var(--v-primary-base) !important;
+
+  .v-icon {
+    color: var(--v-primary-base) !important;
```

Comment:
These are to override the default Vuetify "light theme" colours and match our designs instead.
Maybe in the future we should move these CSS rules to a global file.